### PR TITLE
fix(style): background darkness fixed on hovering disable button

### PR DESCRIPTION
**Pull request by farhan787**
Hover fix issue #3895

### CODE REVIEW

File: packages/fxa-content-server/app/styles/modules/_button-row.scss [hunk 1]
```diff
@@ -73,6 +73,7 @@ button {
 
   &:disabled,
   &.disabled {
+    background: darken($button-background-color-default-hover, 5);
```
Comment (LZoog) [inline packages/fxa-content-server/app/styles/modules/_button-row.scss]: I believe `$button-background-color-default` is the same color as this. Can we use that variable instead for consistency?

Comment (farhan787) [reply packages/fxa-content-server/app/styles/modules/_button-row.scss]: @LZoog yeah, that makes more sense for consistency. Fixed that.

**Comment by dannycoates** [comment]
@farhan787 can you possibly sign this commit?

**Comment by farhan787** [comment]
> @farhan787 can you possibly sign this commit?

oh yeah sure, I forgot to do that...Done.

**Pull request merged**